labanane/frontend/src/components/playlist/youtube-player.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div id='youtube-player'>
+
+  </div>
+</template>
+
+<script>
+
+  export default {
+    name: 'youtube-player',
+    props: ['ready'],
+    data: function () {
+      return {
+        player: ''
+      }
+    },
+    created: function () {
+      this.load_player();
+    },
+    methods: {
+      load_player: function () {
+        var that = this,
```

Comment:
tu cherches à faire quoi ? Le `this` de la méthode est déjà [bindé à l'instance Vue](https://vuejs.org/v2/api/#methods).